lib/helper.js

```diff
@@ -0,0 +1,12 @@
+var semver = require('semver');
+
+exports.DEFAULT_LB_VERSION = '2.x';
+
+/**
+ * validate loopback version
+ * @param version
+ * @returns valid loopback version or default if invalid
+ */
+exports.validateLoopbackVersion = function(version) {
```

Comment:
I think we spell LoopBack with capital `L` and `B`, see e.g. http://loopback.io/ or the caption of https://github.com/strongloop/loopback-workspace/blob/3e80fcd3053199c5de549a5ded5e7773ad2fc5cc/README.md#loopback-workspace-30.

Could you please rename the method to `validateLoopBackVersion`?